src/i18n/fr.json

```diff
@@ -45,6 +45,9 @@
     "lang-control-text-ru": "Русский",
     "lang-control-text-en": "English"
   },
+  "mobile-menu": {
+    "label_to-main-menu": "Retour au menu principal"
```

Comment:
Translations should be added only for Russian and English, other languages ​​via tanker